.github/workflows/suppressions-test.yaml

```diff
@@ -10,14 +10,14 @@ on:
       - package-lock.json
       - package.json
       - tsconfig.json
-      - .github/workflows/_eng-tools-test.yaml
+      - .github/workflows/_template-eng-tools-test.yaml
```

Comment:
I think we are actually confusing terminology here. ADO "workflow templates" map to GH actions "[reusable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows)", while GH actions has its own separate concpet of [workflow templates](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization). 

I am concerned that if we name it `_template` we will confuse people experienced with GH actions.